actionpack/lib/action_dispatch/journey/gtg/builder.rb

```diff
@@ -123,18 +122,9 @@ def lastpos(node)
           end
         end
 
-        def followpos(node)
-          followpos_table[node]
-        end
-
         private
-          def followpos_table
-            @followpos ||= build_followpos
-          end
-
           def build_followpos
-            table = Hash.new { |h, k| h[k] = [] }
-            @ast.each do |n|
+            @ast.each_with_object(Hash.new { |h, k| h[k] = [] }) do |n, table|
```

Comment:
Is `each_with_object` faster than `each`?
This does not look like for performance.

Reply:
Indeed, this one was mostly cosmetic. I'll remove this one.